sdk/python/kfp/components/_structures.py

```diff
@@ -190,15 +190,15 @@ def __init__(self,
 class ContainerSpec(ModelBase):
     '''Describes the container component implementation.'''
     _serialized_names = {
-        'file_outputs': 'fileOutputs', #TODO: rename to something like legacy_unconfigurable_output_paths
+        'unconfigurable_output_paths': 'unconfigurableOutputPaths', #TODO: rename to something like legacy_unconfigurable_output_paths
```

Comment:
could you have comments on the reason of this name? also is the todo still needed?

Reply:
Done.